felix/bpf-gpl/Makefile

```diff
@@ -64,6 +68,20 @@ C_FILES:=tc_preamble.c tc.c connect_balancer.c connect_balancer_v46.c connect_ba
 all: $(OBJS)
 ut-objs: $(UT_OBJS)
 
+libbpf: $(LIBBPF_FILE_CREATED)
+$(LIBBPF_FILE_CREATED):	
+ifeq ("$(wildcard ./$(LIBBPF_DIR))", "")
+	$(info "Directory does not exist.")
+	@echo "Directory does not exist."
+	git clone --depth 1 --single-branch https://github.com/libbpf/libbpf.git
```

Comment:
Is it less network traffic to download release archive directly (say from https://github.com/libbpf/libbpf/archive/refs/tags/v1.4.6.tar.gz)?